tools/driver/driver.cpp

```diff
@@ -61,7 +61,10 @@ extern int modulewrap_main(ArrayRef<const char *> Args, const char *Argv0,
 extern int swift_format_main(ArrayRef<const char *> Args, const char *Argv0,
                              void *MainAddr);
 
-/// Determine if the given invocation should run as a subcommand.
+/// Determine if the given invocation should run as a "subcommand".
+/// Examples of "subcommands" are 'swift build' or 'swift-test', which are
```

Comment:
Nitpick: please include a newline here in case anyone ever does run Doxygen on the Swift source.

Reply:
Also that should be "swift test", right?